jipdate.py

```diff
@@ -128,6 +128,11 @@ def get_parser():
             default=False, \
             help='Load all Jira issues, not just the once marked in progress.')
 
+    parser.add_argument('-r', required=False, action="store_true", \
```

Comment:
This is almost the same as running with only `-q`, but it doesn't spawn the editor and you don't have to say "n". This is actually how one I had it working from the beginning. You want to pipe the output from the script?

Maybe rename the parameter to `-p` as in "Print to stdout"?